app/components/SearchBox.tsx

```diff
@@ -47,10 +47,18 @@ const SearchBox: React.FunctionComponent<Props> = ({
 
   const uiSchema = {
     searchField: {
-      'ui:col-md': 5
+      'ui:col-md': 5,
+      'ui:placeholder': 'Search By:',
+      'ui:options': {
+        label: false
+      }
```

Comment:
The search inputs' `<label>`s were explicitly added as part of a recent accessibility upgrade. Screenreaders etc. cannot comprehend placeholder text as semantically being the label for that input - only the `<label>`

Regardless of the accessibility context, we're trying to lock down v1.x by not unnecessarily introducing UI changes such as this until v2.